projects/1.18/assets/verdure/verdure/lang/zh_cn.json

```
@@ -0,0 +1,30 @@
+{
+    "block.verdure.blue_daisies": "蓝雏菊",
+    "block.verdure.brown_mushroom_shelf": "棕蘑菇丛",
+    "block.verdure.clover": "三叶草",
+    "block.verdure.daisies": "雏菊",
+    "block.verdure.pebbles": "小石堆",
+    "block.verdure.pink_daisies": "粉雏菊",
+    "block.verdure.wildflowers": "野花",
+    "block.verdure.red_mushroom_shelf": "红蘑菇丛",
```

```suggestion
    "block.verdure.blue_daisies": "蓝雏菊",
    "block.verdure.brown_mushroom_shelf": "侧生棕色蘑菇丛",
    "block.verdure.clover": "三叶草",
    "block.verdure.daisies": "雏菊",
    "block.verdure.pebbles": "小石堆",
    "block.verdure.pink_daisies": "粉雏菊",
    "block.verdure.wildflowers": "野花",
    "block.verdure.red_mushroom_shelf": "侧生红色蘑菇丛",
```
注意原版译名

---

```suggestion
    "block.verdure.blue_daisies": "蓝色雏菊",
    "block.verdure.brown_mushroom_shelf": "侧生棕色蘑菇丛",
    "block.verdure.clover": "三叶草",
    "block.verdure.daisies": "雏菊",
    "block.verdure.pebbles": "小石堆",
    "block.verdure.pink_daisies": "粉红色雏菊",
    "block.verdure.wildflowers": "野花",
    "block.verdure.red_mushroom_shelf": "侧生红色蘑菇丛",
```
颜色这部分不强制，如果觉得读起来不通顺就不要改了

---

已修改